central/networkgraph/entity/gatherer/online_gatherer.go

```diff
@@ -143,3 +152,31 @@ func (g *defaultExtSrcsGathererImpl) reconcileDefaultExternalSrcs() error {
 	}
 	return nil
 }
+
+// loadLocalChecksum loads local checksum if it exists.
+func (g *defaultExtSrcsGathererImpl) loadLocalChecksum(store blobstore.Datastore) ([]byte, error) {
+	if len(g.currentChecksum) > 0 {
+		return g.currentChecksum, nil
+	}
+	buf, _, exists, err := store.GetBlobWithDataInBuffer(blobAccessReadCtx, defaultexternalsrcs.LocalChecksumBlobPath)
+	if err != nil || !exists {
+		return nil, err
+	}
+	g.currentChecksum = buf.Bytes()
+	return g.currentChecksum, nil
+}
+
+func (g *defaultExtSrcsGathererImpl) writeChecksumLocally(store blobstore.Datastore, checksum []byte) error {
```

Comment:
I find this name confusing.  Looks to me that we are writing the checksum we found locally to the blobstore.  the name implies (at least to me) that we are writing it to the local file system.